payments_service/auth/views.py

```diff
@@ -26,6 +26,7 @@ def post(self, request):
 
         api = solitude.api()
         created = False
+        locale = request.META.get('HTTP_ACCEPT_LANGUAGE')
```

Comment:
if this is empty or None, should we still patch Solitude? I'm leaning towards no since that could overwrite an existing value. I'm not sure why this header might be empty but it seems possible on an obscure mobile browser or some other client we don't know about it.